lib/context.js

```diff
@@ -437,6 +443,15 @@ api.createTermDefinition = ({
         'absolute IRI or a blank node identifier.',
         'jsonld.SyntaxError', {code: 'invalid IRI mapping', context: localCtx});
     }
+
+    if(reverse.match(KEYWORD_PATTERN)) {
+      // FIXME: remove logging and use a handler
+      console.warn('WARNING: values beginning with "@" are reserved' +
+        ' for future use and ignored', {reverse});
+      activeCtx.mappings.delete(term);
```

Comment:
In the case of warnings (here and below), should this restore the `previousMapping` for `term` instead of deleting it?

Reply:
Yes, it should restore before returning; I'll take care of that.

Reply:
Fixed in 649857c33c42774640b51b83ed1e294f7d5a2b92.